test/common/stats/thread_local_store_test.cc

```diff
@@ -80,6 +80,19 @@ class StatsThreadLocalStoreTest : public testing::Test, public RawStatDataAlloca
     store_->addSink(sink_);
   }
 
+  void setUp() {
```

Comment:
void SetUp() override { ....

then you can remove the explicit calls from the test methods.  It has to be applicable to all the test-methods in the class, so you might want to make a subclass just for the histograms.

Reply:
I have a added a new sub class and made this override and removed explicit calls. similar change for TearDown.